### ISSUE TITLE
Bitcoin Lake Guatemala

### ISSUE THREAD

**Issue by dadofsambonzuki** [teambtcmap/btcmap-data#34]
Here is the [KML script HTML](https://data.btcmap.org/elements-guatemala.html) for easier entry into OSM.

All links open an OSM editor centred on the location. Use the button zoom to pinpoint. Edit an existing location if present, create a new one if not.

Have twitter DM-ed the team there for them to work through it.

**Comment by jmann6** [comment]
I've been slowly working through these.  Thanks for KML script! 
![Bitcoin lake map](https://user-images.githubusercontent.com/3744618/193408544-71bebaf8-deb5-4627-bc79-a05362e12509.PNG)

**Comment by dadofsambonzuki** [comment]
Awesome work!

We need to get you setup with a community page, e.g. btcmap.org/community/bitcoin-lake-guatemala?

Also, I'll add you to the Shadowy Supertagger team so you can manage issues.

**Comment by jmann6** [comment]
Yes, please add me to the Shadowy Supertagger team.  A community page, e.g. btcmap.org/community/bitcoin-lake-guatemala would be great.

**Comment by jmann6** [comment]
For reference:  https://twitter.com/LakeBitcoin/status/1573355853360537600?s=20&t=Yh849_T0u1hGv05ijCx0fA

**Comment by dadofsambonzuki** [comment]
How are you progressing on this @jmann6? Need some help or keeping the points all to yourself?!

**Comment by jmann6** [comment]
I added all the pins around Bitcoin Lake a few of the IBEX Mecardo locations; however, there are a lot of pins in Guatemala City and throughout Guatemala that need to be added.  I'll be sure to verify some of these places in-person during the Bitcoin Lake bitcoin trip after Adopting Bitcoin 2022.  Anyone is free to jump in and help!  :)

I've been focusing on Colorado, where I live, for now.

**Comment by dadofsambonzuki** [comment]
Hey @jmann6 - is there a way for you to say which ones from this list you've done?

We can split the rest out into individual Issues so we can more effectively work through them as a team.

**Comment by bubelov** [comment]
@dadofsambonzuki @jmann6 any updates on that issue?

**Comment by jmann6** [comment]
I updated a few locations after visiting Bitcoin Lake, but haven't focused on Guatemala for a while.

**Comment by dadofsambonzuki** [comment]
I'll contact the community leads there and hopefully have them go through the list and organise locally.

**Comment by dadofsambonzuki** [comment]
Hmm, the remaining ones are not specific to Bitcoin Lake and so not really for them.

The rest need working through and verifying.

@jmann6 - how is it best to figure out which ones you've added? Was there a method to the additions?!

**Comment by jmann6** [comment]
@dadofsambonzuki The OSM data in Central America is not great (buildings not marked etc and difficult to pin point exact location) so I focused on only the locations around Lago de Atitlan.  I tried to get most of these.
![bitcoin lake](https://user-images.githubusercontent.com/3744618/215549855-9bd9308c-b11c-45cb-88f3-0f17d5414003.PNG)

However, there are a lot more BitcoinBeachWallet (BBW) locations throughout Guatemala that need added.
![bitcoin lake2](https://user-images.githubusercontent.com/3744618/215550123-6bbeb558-74e9-4e9c-b823-d4717a622f73.PNG)
![Screenshot_20230130-101704](https://user-images.githubusercontent.com/3744618/215550723-b23dd5df-6345-4430-bbce-1243144524dd.png)

Google map reference for Bitcoin Lake:  https://www.google.com/maps/d/u/0/viewer?hl=en&hl=en&ll=14.530059316336521%2C-91.34372270831372&z=10&mid=1tUXjh07rEof5bHpyO8BBcYF2gUuumwuT

Might be easier to close this issue and create a new issue focused on Guatemala City or Guatemala as a whole?  That KML script is most likely out-of-date and probably need a new one as well.
